FogCreek/Glitch-Community
# Profuse tea: Update styled components to ssr without vendoring

## Links
https://profuse-tea.glitch.me/

## Changes:
- Removes src/components/shared-components.js
- Cleaned up the aliases and added extra error logging
- Remove a long defunct sideEffect entry from package.json

## How To Test:
Try using the site and see if any ssr pages look wrong or if there are any react hydration warnings

## Feedback I'm looking for:
I'm not certain that the styled-components babel plugin is running everywhere it should, but there aren't any hydration errors I can find
